### DISCUSSION TITLE
Roadmap ??

### DISCUSSION THREAD

**Discussion by brianteeman** [joomla/joomla-cms#38043] [General]
The published [roadmap ](https://developer.joomla.org/roadmap.html) says that we are using semantic versioning and not time based releases. However my understanding is that we are using time based releases. There is also no mention on the roadmap of anything after 4.1.

There was however a magazine article for [Joomla 5](https://magazine.joomla.org/all-issues/january-2022/joomla-5-is-in-the-planning-meet-the-release-leads) back in January.

In January the production department [reported ](https://volunteers.joomla.org/departments/production/reports/1678-production-dept-meeting-minutes-january-11-2022)"Created a Joomla 5 Roadmap document based on Harald's brainstorming list. Suggestions for changes and additions have already been incorporated as far as possible. The next step is to prioritise the individual points; a Voting Sheet for this has been prepared."

February - production department [report in Feb](https://volunteers.joomla.org/departments/production/reports/1699-production-dept-meeting-minutes-february-22-2022) stated **A Google Form will be provided to collect input from the wider community**

March - the production department [reported ](https://volunteers.joomla.org/departments/production/reports/1712-production-dept-meeting-minutes-march-22-2022) "Niels and Harald are meeting on Thursday _(march 25_) to discuss the preliminary roadmap for Joomla 5. 

There was also mention of a specific date in 2023 for the release of J5 but I cant find the reference now.

What is the current status?

**Comment by nzyeehaa** [comment]
@brianteeman 
harald... Based on this we plan to release Joomla! 5.0 on 17. August 2023....

neils... unless the rules of the game change before then, Joomla 5 will be released on 17 August 2023...

https://magazine.joomla.org/all-issues/january-2022/joomla-5-is-in-the-planning-meet-the-release-leads
